docs/examples/tutorial/cython_tutorial/primes.pyx

```diff
@@ -18,6 +22,6 @@ def primes(int nb_primes):
             len_p += 1
         n += 1
 
-    # Let's return the result in a python list:
+    # Let's put the result in a python list:
```

Comment:
See above about better ways to say this.

Reply:
resolved [here](https://github.com/cython/cython/pull/4226/commits/64f9e0adb2604de2f789a24e47fc23ae74b0186b)